mappings/servers/twerion/metadata.json

```diff
@@ -0,0 +1,64 @@
+{
+    "id": "twerion",
+    "name": "Twerion Network",
+    "description": "A network focused on PvP",
+    "addresses": [
+        "twerion.net",
+        "twerion.de",
+        "twerion.com",
+        "twerion.eu",
+        "twerion.org"
+    ],
+    "primaryAddress": "twerion.net",
+    "minecraftVersions": [
+        "1.8.*"
+    ],
+    "primaryMinecraftVersion": "1.8.9",
+    "primaryColor": "#FFAA00",
+    "secondaryColor": "#FF8000",
+    "primaryLanguage": "en",
+    "languages": [
+        "en",
+        "es-EN",
+        "it",
+        "tr",
+        "ar",
+        "zh-CN",
+        "nl",
+        "fr",
+        "de",
+        "el",
+        "ja",
+        "ko",
+        "lt",
+        "no",
+        "pl",
+        "pt-PT",
+        "pt-BR",
+        "sv-SE"
+    ],
```

Comment:
Something tells me you don't actually support these languages and might be listing for the sake of listing. Please ensure you're keeping this list accurate. We only accept languages that are truly supported.

![image](https://github.com/user-attachments/assets/b378ab94-2b60-430c-b969-55eef61c6b6c)

Reply:
Why should we add 30 website translations when every browser has an auto translator built in?

Just do /language on the twerion.net server.

![image](https://github.com/user-attachments/assets/f71d0de5-a36e-4c4c-b66b-e6fd62ba7c71)

Reply:
Alright, if that’s there, I’ll accept it.